Gemfile

```
@@ -76,6 +76,7 @@ gem 'mediaelement-title', git:'https://github.com/avalonmediasystem/mediaelement
 gem 'mediaelement-track-scrubber', git: 'https://github.com/avalonmediasystem/mediaelement-track-scrubber.git', tag: 'avalon-r6'
 
 # Jobs
+gem 'redis', '~> 3.3.5'
 gem 'redis-rails'
 gem 'resque', '~> 1.27.0'
 gem 'resque-scheduler', '~> 4.3.0'
```

> The redis >=4.0 gem causes an error 'NoMethodError: undefined method for Redis:Class'
> 
> My though, pin the redis gem and add a technical debt issue? Thoughts?

Redis was bumped by resque-scheduler (https://github.com/ualbertalib/avalon/pull/429/commits/0095408e4c784eba07f7f019e06d7b128afbb841#diff-e79a60dc6b85309ae70a6ea8261eaf95R707).  Alternately could pin that gem until that problem has been resolved upstream.

```suggestion
gem 'resque-scheduler', '4.3.0'
```